data/data/powervs/main.tf

```diff
@@ -4,21 +4,60 @@ provider "ibm" {
   zone             = var.ibmcloud_zone
 }
 
+data "ibm_resource_group" "cos_group" {
+  name = var.powervs_resource_group_name
+}
+
+resource "ibm_resource_instance" "cos_instance" {
+  name              = var.powervs_cos_instance_name
```

Comment:
this can be a autogenerated one based on the clusterid

Reply:
good idea! Done.